### PR TITLE
Use the acct: URI for this domain, if in h-card

### LINKED ISSUE #3: allow users to choose their own username
I know it's standard IndieWeb practice to have only one user on the domain, but that doesn't necessarily mean their name is "me", and it's misleading to return (for example) WebFinger data that claims it's for `acct:me@example.com` when that address doesn't exist.

For example, on my site, I've provided an email address in [my representative h-card](https://indiewebify.me/validate-h-card/?url=https%3A%2F%2F00dani.me). When this is done, I think it would be appropriate for Bridgy Fed to use that email address to construct the `acct:` identifier for the user.

### PR DESCRIPTION
**Pull request by singpolyma**
This allows users to choose their own username for their webfinger ID.

Closes #3

### CODE REVIEW

File: webfinger.py [hunk 1]
```diff
@@ -58,13 +58,20 @@ def template_vars(self, domain, url=None):
...
         props = hcard.get('properties', {})
         urls = util.dedupe_urls(props.get('url', []) + [resp.url])
         canonical_url = urls[0]
 
+        acct = '%s@%s' % (domain, domain)
+        for url in urls:
+            if url.startswith('acct:'):
+                urluser, urldomain = util.parse_acct_uri(url)
+                if urldomain == domain:
+                    acct = '%s@%s' % (urluser, domain)
```
Comment (snarfed) [inline webfinger.py]: mind logging here that we're use this overridden `acct:` url?

Comment (singpolyma) [reply webfinger.py]: done

**Comment by snarfed** [comment]
thanks again for the contribution!

**Comment by snarfed** [comment]
deployed.

**Pull request merged**